packages/amplify-category-auth/resources/schemas/cognito/ServiceQuestionsResult.schema.json

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
Is this file auto-generated?

Reply:
yes. This will be autogenerated with every release  using ```generateSchemas``` and will be shipped with new release